packages/tool-cache/src/tool-cache.ts

```diff
@@ -627,8 +627,11 @@ export async function getManifestFromRepo(
       core.debug('Invalid json')
     }
   }
-
-  return releases
+  if (!releases.hasOwnProperty('documentation_url')) {
```

Comment:
👋 Perhaps we could check for the rate-limiting response code (429) and return an error instead of relying on the presence of the `documentation_url` property?